precompiles/ics20/approve_test.go

```diff
@@ -137,7 +137,7 @@ func (s *PrecompileTestSuite) TestApprove() {
 			func() []interface{} {
 				return []interface{}{
 					s.address,
-					[]ics20.Allocation{
+					[]cmn.Allocation{
```

Comment:
TBH I think this makes it less verbose, as you can see in the old version, it was clear that it was an `ics20.Allocation`, now you have to know that this has to do with ICS20. Maybe it makes more sense to call it `ICS20Allocation` if it lives in the general types now?